addons/gunbag/functions/fnc_toGunbagCallback.sqf

```diff
@@ -40,6 +43,6 @@ _unit removeWeapon _weapon;
 _gunbag setVariable [QGVAR(gunbagWeapon), [_weapon, _items, _magazines], true];
 
 // play sound
-if (!isNil "ACE_Backpacks") then {
+if (["ace_backpacks"] call EFUNC(common,isModLoaded)) then {
```

Comment:
>  If it's not needed, why do we have that function in the first place?

Actually it comes from CSE. I believe CBA creates the `ACE_Backpacks` variable to signal that a CfgPatch with that name was loaded, so both methods serve the same purpose.